api/base/v1alpha1/datasource_types.go

```diff
@@ -38,10 +38,12 @@ type DatasourceSpec struct {
 }
 
 type RDMA struct {
-	// Path on a model storage server, the usual storage path is /path/ns/mode-name, and the path field is /path/, which must end in /.
+	// ServerSavePath on a model storage server, the usual storage path is /path/ns/mode-name, and the path field is /path/, which must end in /.
 	// example: /opt/kubeagi/, /opt/, /
 	// +kubebuilder:validation:Pattern=(^\/$)|(^\/[a-zA-Z0-9\_.@-]+(\/[a-zA-Z0-9\_.@-]+)*\/$)
-	Path string `json:"path"`
+	ServerSavePath string `json:"path"`
```

Comment:
ServerPath string `json:"serverPath"`

Reply:
How about `Path string`?
This path is the path of the sender on the one hand, and the path of the receiver on the other.